root/static/css/style.css

```diff
@@ -1396,6 +1396,14 @@ a.curs-editable-table-field img, a.curs-editable-table-long-field img
   display: inline;
 }
 
+.curs-annotation-edit .curs-edit-wide-field {
+  width: 100%;
+}
+
+.curs-annotation-edit .curs-dialog-body-contents {
+  overflow: hidden;
```

Comment:
Setting `overflow` is a good idea. Only thing I'd suggest is using `auto` instead of `hidden`, since even though it adds a horizontal scrollbar at small screen sizes, it might be better to let users have some way to see the clipped content.

Reply:
auto is a good idea.  I just tried it but the scrollbar doesn't disappear when the page it full size.  I can't see what the problem is.  Do you have any idea?